docs/setup.rst

```diff
@@ -94,18 +95,97 @@ The following sub-sections provide guidance on how to install the recommended to
 - **Linux** TODO
 - **macOS** TODO
 
+.. _setup_zlib:
+
+zlib
+----
+:zlib:`zlib<>` is a common compression library.
+It's important to use a 64-bit version of the library as the 32-bit is not compatible with libCellML.
```

Comment:
Replace occurrences of `[Ii]t's` with `[Ii]t is`.

Reply:
e1b924d

Reply:
`It's` should still be replaced with `It is`.